extensions/amp-analytics/0.1/vendors/permutive.js

```diff
@@ -31,14 +31,15 @@ export const PERMUTIVE_CONFIG = /** @type {!JsonObject} */ ({
       '&_ep_geo_info=%24ip_geo_info',
     'engagement': '${track}&e=PageviewEngagement&_ep_engaged_time=5',
     'completion': '${track}&e=PageviewEngagement&_ep_completion=0.25',
+    'custom': '${track}&e=${event}',
   },
   'triggers': {
     'trackPageview': {
       'on': 'visible',
       'request': 'pageview',
     },
     'trackEngagement': {
-      'on': 'visible',
+      'on': 'timer',
```

Comment:
@joshfg You understand the default trigger the best. One thing to confirm with you is that by doing this, the number of request will increase significantly on your side. We do want to trigger engagement event at every 5 sec by default without opt-in? 
Thank you!

Reply:
Thanks @zhouyx!
Yes we do expect to see a lot more requests coming from AMP with this change. However it was the intended behaviour originally (the `'on': 'visible'` in the timer trigger was a mistake!) and we track engagement in the same way on standard web pages, so this brings our amp-analytics config in line with how we function elsewhere.

Reply:
Sounds good! Thanks for confirming.